doc/classes/TextureArray.xml

```diff
@@ -4,7 +4,20 @@
 		Array of textures stored in a single primitive.
 	</brief_description>
 	<description>
-		[TextureArray]s store an array of images in a single [Texture] primitive. Each layer of the texture array has its own mipmap chain. This makes it is a good alternative to texture atlases.
+		[TextureArray]s store an array of [Image]s in a single [Texture] primitive. Each layer of the texture array has its own mipmap chain. This makes it is a good alternative to texture atlases.
+		
+		[TextureArray]s must be displayed using shaders. After importing your file as a [TextureArray] and setting the appropriate Horizontal and Vertical Slices, display it by setting it as a uniform to a shader, for example:
+		[codeblock]
+		shader_type canvas_item;
+		
+		uniform sampler2DArray tex;
+		uniform int index;
+		
+		void fragment() {
+    		    COLOR = texture(tex, vec3(UV.x, UV.y, float(index) ));
```

Comment:
```suggestion
    		    COLOR = texture(tex, vec3(UV.x, UV.y, float(index)));
```

Reply:
You should also correct the indentation, currently it's this:

![Screenshot_20210419_124811](https://user-images.githubusercontent.com/4701338/115224720-813f7b00-a10d-11eb-9d4d-d7473d9626cb.png)

It should be two tabs for the XML content indentation, and then 4-spaces for the actual code indentation of the `fragment()` function.